### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

File: frontend/src/components/EditProfile/ContainerEditProfile.jsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default function ContainerEditProfile() {
         </Route>
       </Switch>
       <div className="submit">
-        <button onClick={onSubmit}>Submit your modifications</button>
+        <button onClick={onSubmit}>Valides tes modifications</button>
```
Comment (LoicBrassart) [inline frontend/src/components/EditProfile/ContainerEditProfile.jsx]: No "s" in imperative form

File: frontend/src/components/Discover.jsx [hunk 2]
```diff
@@ -22,29 +23,52 @@ function Discover() {
...
-          user_avatar={post.user_avatar}
-          id={post.id}
-        />
-      ))}
+      <div className="feed">
+        {posts.map(post => (
+          <Postcard
+            key={post.message}
+            message={post.message}
+            tags={post.tags ? "#" + post.tags.split(" ").join(" #") : null}
```
Comment (LoicBrassart) [inline frontend/src/components/Discover.jsx]: `"#" + post.tags.replace(" "," #")` would be better, perf wise

File: frontend/src/components/EditProfile/Role.jsx [hunk 3]
```diff
@@ -26,18 +23,18 @@ const Role = () => {
               });
             }}
           >
-            <option value="">-- Please choose your role --</option>
-            <option value="Player">Player</option>
-            <option value="Pro-player">Pro-player</option>
+            <option value="">-- Séléctionnes ton rôle --</option>
```
Comment (LoicBrassart) [inline frontend/src/components/EditProfile/Role.jsx]: No "s" there

File: frontend/src/components/Form.jsx [hunk 4]
```diff
@@ -56,32 +56,46 @@ function Form() {
...
-            hideProgressBar={true}
-          />
+          <div className="buttonContainer">
+            {page !== 1 ? (
+              <button className="previous" onClick={() => setPage(page - 1)}>
+                Précédent
+              </button>
+            ) : (
+              ""
+            )}
```
Comment (LoicBrassart) [inline frontend/src/components/Form.jsx]:
```
{page !== 1 && 
              <button className="previous" onClick={() => setPage(page - 1)}>
                Précédent
              </button>
}
``` would be better, readability-wise

File: frontend/src/components/Form.jsx [hunk 5]
```diff
@@ -56,32 +56,46 @@ function Form() {
...
+            {page === 2 ? (
```
Comment (LoicBrassart) [inline frontend/src/components/Form.jsx]: Same here

File: frontend/src/components/Form.jsx [hunk 6]
```diff
@@ -135,14 +149,17 @@ function Form() {
           {page === 2 ? (
             <div className="page2">
               <div className="introText">
-                Choisis les jeux pour lesquels tu souhaites avoir des news
-                (penses à scroller):
+                Choisis tes jeux favoris (penses à scroller) :
```
Comment (LoicBrassart) [inline frontend/src/components/Form.jsx]: No "s" to "pense" here

File: frontend/src/components/GameChoice.jsx [hunk 7]
```diff
@@ -6,9 +6,13 @@ function GameChoice(props) {
   return (
     <div className="gameChoice" onClick={() => setClick(!click)}>
       <button className={click ? "status-on" : "status-off"}>
-        <img className="picture" src={props.picture} />
+        <img className="picture" alt="image_jeu" src={props.picture} />
         {click && (
-          <img className="checkbox" src="/pictures/checkboxpurple.png" />
+          <img
+            className="checkbox"
+            alt="jeu_séléctionné"
```
Comment (LoicBrassart) [inline frontend/src/components/GameChoice.jsx]: That's an `alt`, why the underscore ?

File: frontend/src/components/LiveCard.jsx [hunk 8]
```diff
@@ -17,26 +17,23 @@ function LiveCard(props) {
   }, []);
 
   return (
-    <div className="streamingStatus" onClick={toggle}>
+    <div className="streamingStatus" onClick={toggle} title={streamTitle}>
       <Modal isShowing={isShowing} hide={toggle} user_name={props.user_name} />
       {gameId.length > 0 ? (
         <img
           className="blank_game_logo"
+          alt="Logo_Jeu"
```
Comment (LoicBrassart) [inline frontend/src/components/LiveCard.jsx]: Why "_" in an `alt` ?

File: frontend/src/components/LiveCard.jsx [hunk 9]
```diff
@@ -17,26 +17,23 @@ function LiveCard(props) {
...
-          <span className="streamerName">{props.streamer_name}</span> -{" "}
-          {streamTitle.length > 15 ? (
-            <span title={streamTitle}>{streamTitle.slice(0, 12) + "..."} </span>
-          ) : (
-            streamTitle
-          )}
-        </div>
+      <img
+        src="https://www.stickpng.com/assets/images/580b57fcd9996e24bc43c540.png"
+        alt="Image_Stream"
```
Comment (LoicBrassart) [inline frontend/src/components/LiveCard.jsx]: Same

File: frontend/src/components/LiveContainer.jsx [hunk 10]
```diff
@@ -49,8 +49,9 @@ function LiveContainer() {
       </div>
       <div className="liveContent">
         {lives.length > 0
```
Comment (LoicBrassart) [inline frontend/src/components/LiveContainer.jsx]: Useless test: a `map` on an empty array just executes... 0 time

File: frontend/src/components/Modal.jsx [hunk 11]
```diff
@@ -28,8 +28,9 @@ const Modal = ({ isShowing, hide, user_name }) =>
           </div>
           <div className="StreamPlayer">
             <iframe
+              title="Stream_Player"
               src={`https://player.twitch.tv/?channel=${user_name}`}
-              frameborder="0"
+              frameBorder="0"
```
Comment (LoicBrassart) [inline frontend/src/components/Modal.jsx]: CSS ? No ?
`iframe { border: none; }`

File: frontend/src/components/NavBar.jsx [hunk 12]
```diff
@@ -1,17 +1,44 @@
...
   const [burgerMenu, setBurgerMenu] = useState(false);
   const dispatch = useDispatch();
   const user_id = useSelector(state => state.user_id);
+  const history = useHistory();
+  const [search, setSearch] = useState("");
+  const handleSubmit = e => {
+    e.preventDefault();
+    axios
+      .get(`${backend}/api/search/users/?pseudo=${search}`)
+      .then(function(response) {
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx]: Use an arrow function, for consistency's sake

File: frontend/src/components/NavBar.jsx [hunk 13]
```diff
@@ -21,7 +48,34 @@ function NavBar() {
...
+                <input
+                  className="formSearchBar"
+                  type="text"
+                  placeholder="Cherche un utilisateur avec son pseudo"
+                  value={search}
+                  onChange={e => change(e)}
+                />
+              </form>
+            </div>
+          ) : null}
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx]: Use `&&` operator

File: frontend/src/components/NavBar.jsx [hunk 14]
```diff
@@ -42,23 +96,30 @@ function NavBar() {
           {displayMenu ? (
             <img
               className="triangle"
+              alt=">"
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx]: I think "Next" would be easier to read for blind users

File: frontend/src/components/NavBar.jsx [hunk 15]
```diff
@@ -69,51 +130,67 @@ function NavBar() {
...
-                <span className="FirstLetter">D</span>éconnection
-              </li>
+              <ul>
+                <li>
+                  <Link to={`/userpage/${user_id}`}>
+                    <span className="FirstLetter">P</span>rofil
+                  </Link>
+                </li>
+                <li onClick={() => dispatch({ type: "DISCONNECT" })}>
+                  <span className="FirstLetter">D</span>éconnection
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx]: "-ct +x" in French

File: frontend/src/components/NewsFeed.jsx [hunk 16]
```diff
@@ -57,25 +57,35 @@ function NewsFeed() {
...
+                  statuslike={post.liked}
+                  userPseudo={post.pseudo}
+                  userTeam={post.team_name}
+                  key={post.id}
+                />
+              ))
+          : posts.map(post => (
               <Postcard
                 message={post.message}
+                tags={post.tags ? "#" + post.tags.split(" ").join(" #") : null}
```
Comment (LoicBrassart) [inline frontend/src/components/NewsFeed.jsx]: See other comment

File: frontend/src/components/NewsFeed.jsx [hunk 17]
```diff
@@ -57,25 +57,35 @@ function NewsFeed() {
...
-      ))}
-      <Filter />
-      {filters.length > 0
-        ? posts
-            .filter(post => filters.includes(post.game_id))
-            .map(post => (
+      <div className="feed">
+        <PostField />
+        <Filter />
+        {filters.length > 0
```
Comment (LoicBrassart) [inline frontend/src/components/NewsFeed.jsx:63]: See other comment

File: frontend/src/components/SignIn.jsx [hunk 18]
```diff
@@ -39,24 +39,22 @@ const SignIn = () => {
...
+        <h2>Tu as un compte ?</h2>
         <form
           onSubmit={e => {
             e.preventDefault();
           }}
         >
           {!credentialsState && (
             <p className="wrong">
-              vos informations semblent fausses, veuillez réesayer
+              tes informations semblent fausses, réessaies.
```
Comment (LoicBrassart) [inline frontend/src/components/SignIn.jsx]: - Capital letter when beginning a sentence
- No "s" to "réessaie" here

File: frontend/src/components/Tag.jsx [hunk 19]
```diff
@@ -18,7 +16,7 @@ function Tag() {
   return (
     <div className="tag">
       <ReactTagInput
-        placeholder="Tapez et appuyez sur entrée pour ajouter un tag"
+        placeholder="Tapes et appuies sur entrée pour ajouter un tag"
```
Comment (LoicBrassart) [inline frontend/src/components/Tag.jsx]: Nope, and nope

File: frontend/src/components/UserPage.jsx [hunk 20]
```diff
@@ -67,40 +67,45 @@ export default function UserPage() {
...
-            )}
+            <p>Ma description : {user.bio}</p>
           </div>
         </div>
       ) : null}
       <div className="posts">
         {posts.map(post => (
           <Postcard
             message={post.message}
+            tags={post.tags ? "#" + post.tags.split(" ").join(" #") : null}
```
Comment (LoicBrassart) [inline frontend/src/components/UserPage.jsx]: See other comment

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
This PR is full of bad practices:
- Too many impacted files (54)
- Too many commits, spread on a too long period (why didn't you do any PR sooner ?)
- Too many people involved (not a real bad practice, per se... When working with seasoned developers. Here, I'm not sure everyone worked with the same guidelines)

I didn't proofread the CSS, I don't have enough time left aahead of me, sorry about that.

**Review by Gumatan — APPROVED** [review]

File: frontend/src/components/LiveContainer.jsx [hunk 21]
```diff
@@ -48,18 +48,17 @@ function LiveContainer() {
...
-                streamer_name={live.user_name}
-                stream_title={live.title}
-                viewer_count={live.viewer_count}
-                user_name={live.user_name}
-                game_id={live.game_id}
-              />
-            ))
-          : null}
+        {lives &&
+          lives.map((live) => (
```
Comment (LoicBrassart) [inline frontend/src/components/LiveContainer.jsx:52]: Not what I had in mind, but nevermind.

**Review by LoicBrassart — APPROVED** [review]
Even though you don't find it funny, it would be appreciated if you were to write full sentences for your commit messages. Please look the commit list aloud and see if you'll know what you did, six months from now ?

I didn't see any new Evil, though, let's go

**Pull request merged**